internal/pkg/cli/task_run.go

```diff
@@ -129,6 +152,61 @@ func (o *runTaskOpts) validateEnvName() error {
 	return nil
 }
 
+func (o *runTaskOpts) askTaskFamilyName() error {
+	if o.familyName != "" {
+		return nil
+	}
+
+	// TODO during Execute: list existing tasks like in ListApplications, ask whether to use existing tasks
+
+	familyName, err := o.prompt.Get(
+		fmtTaskRunFamilyNamePrompt,
+		taskRunFamilyNamePromptHelp,
+		validateTaskFamilyName,
+		prompt.WithFinalMessage("Task family name:"))
+	if err != nil {
+		return fmt.Errorf("prompt get task family name: %w", err)
+	}
+	o.familyName = familyName
+	return nil
+}
+
+func (o *runTaskOpts) askEnvName() error {
+	if o.env != "" {
+		return nil
+	}
+
+	if o.AppName() == "" {
```

Comment:
AppName() could still be "" here when the `task run` command isn't run under a workspace and the user didn't provide an app name. Unlike `svc init`, `task run` doesn't have to be under a workspace, in which case we will just use the default VPC and cluster instead of an environment previously created by Copilot (hence the envNameNone)